app/services/additional_claimants_csv/validator.rb

```diff
@@ -1,3 +1,6 @@
+class CsvDobValidationError < StandardError
```

Comment:
maybe we could even move this into the `AdditionalClaimantsCsv::Validator` class:

```ruby
class AdditionalClaimantsCsv::Validator < AdditionalClaimantsCsv::BaseCsv
  class CsvDobValidationError < StandardError; end
  .
  .
  .
end
```

what do you think?